content/hi/docs/contribute/_index.md

```diff
@@ -0,0 +1,177 @@
+class A,B,C,D,E,F,G,H,M,Q,N,O,P,V grey
+class S,T,U spacewhite
+class first,second,third white
+{{</ mermaid >}}
+***आकृति  - नए योगदानकर्ताओ के लिए योगदान शुरू करने का रास्ता***
+
+ऊपर  दी गई आकर्ति नए योगदानकर्ता के लिए दिशानिर्देश'है।  साइन अप या समीक्षा के लिए आप इनमे से कुछ निर्देश या सभी निर्देश का पालन कर सकते है ।  अब आप PR करने के लिए तैयार है  जो आपके योगदान उद्देश को पूरा करे , उन मे से कुछ Open PR खंड मे है । एक बार फिर से आपके सभी प्रशनों का स्वागत है । 
+
+कुबेरनेटेस  समुदाय मे कुछ काम मे ज्यादा विश्वास और एक्सेस की जरूरत पड़ती है। भूमिका और अनुमति के बारे मे ज्यादा जानकारी के लिए  
+ [सिग डॉक्स मे भाग लेना ](/docs/contribute/participate/) को जरूर देखे । 
+
+## आपका पहला योगदान 
+
+आप अपने पहले योगदान की तैयारी के लिए दिए गए दिशानिर्देश को देख ले। नीचे दिया हुआ चित्र दिशानिर्देश और उसकी विस्तार मे जानकारी देता है। 
+
+<!-- See https://github.com/kubernetes/website/issues/28808 for live-editor URL to this figure -->
+<!-- You can also cut/paste the mermaid code into the live editor at https://mermaid-js.github.io/mermaid-live-editor to play around with it -->
+
+{{< mermaid >}}
+flowchart LR
+    subgraph second[पहला योगदान]
+    direction TB
+    S[ ] -.-
+    G[दूसरे K8s मेम्बर्स के <br> prs की समीक्षा करे] -->
+    A[पहली अच्छी समस्या के लिए K8s/website<br>की समस्या सूची पर जाए ] --> B[pr ओपन करे!!]
+    end
+    subgraph first[सूचित तैयारी ]
+    direction TB
+       T[ ] -.-
+       D[योगदान अवलोकन को पढे] -->E[K8s content<br>and style guide को पढे]
+       E --> F[Hugo page<br>content types<br>और shortcodes के बारे मे जाने]
+    end
+    
+
+    first ----> second
+     
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,D,E,F,G grey
+class S,T spacewhite
+class first,second white
+{{</ mermaid >}}
+***आकृति  - आपके पहले योगदान की तैयारी ***
+
+- योगदान करने के विभिन्न तरीकों को जानने के लिए  [योगदानकर्ता अवलोकन](/docs/contribute/new-content/overview/) को पढे । 
+- अच्छे प्रवेश बिन्दु के लिए  [`कुबेरनेटेस /वेबसाईट ` मुद्दा सूची ](https://github.com/kubernetes/website/issues/)
+  को जाँचे । 
+- [गिट हब का प्रयोग करते हुए पुल रीक्वेस्ट खोले  ](/docs/contribute/new-content/open-a-pr/#changes-using-github) पुराने प्रलेखन मे और मुद्दा को भरने की प्रक्रिया के बारे मे ज्यादा अच्छे से जाने । 
+- भाषा  और ठीक तरीके से पुल रीक्वेस्ट के लिए  दूसरे कुबेरनेटेस समुदाय के सदस्यों के  [पुल रीक्वेस्ट की समीक्षा करे ](/docs/contribute/review/reviewing-prs/) । 
+- कुबेरनेटेस  [प्रकरण ](/docs/contribute/style/content-guide/) और  
+  [स्टाइल मार्गदर्शक](/docs/contribute/style/style-guide/) को पढे ताकि आप सूचित टिप्पणी दे सके। 
+- [पेज प्रलेखन टाइप ](/docs/contribute/style/page-content-types/)
+  और  [हुगो शॉर्टकोडेस ](/docs/contribute/style/hugo-shortcodes/) के बारे मे जाने। 
```

Comment:
Watch out for machine translation. Some of these terms need to stay in the original.